### ISSUE TITLE
New Hardware Revision Controller Board 032-1.0

### ISSUE THREAD

**Issue by rsp-git** [Emile666/stc1000_stm8#19]
Hello,
i bricked my old one with 032-0.0 and ordered a new one.
Now its without SWIM Connector and now there are 2 ICs under the Display.
![WhatsApp Image 2024-10-07 at 15 19 29](https://github.com/user-attachments/assets/c6542be1-41f5-4ff8-8fbd-eed0e6dd6f9c)
![WhatsApp Image 2024-10-07 at 15 19 30](https://github.com/user-attachments/assets/8c840077-abd4-49ab-bf1f-586dbfe2eb4e)

Do you have an Idea if this is still a STM8? The ICs are cleaned.

BR

**Comment by Emile666** [comment]
I don't think so. This is probably one of these Chinese 8051 clones, that they use a lot nowawadays. The display is also interesting, only 7 pins? I am probably missing something here. Also the solder connector is quite different from the original stc1000. What  I typically do is replace both PCBs with custom ones including an STM8s105. And I reuse components like casing and display. But that will probably not work here, since the display is too different.

**Comment by gzalo** [comment]
The display seems to be charliplexed or something similar, with 7 pins it should be able to control up to 42 LEDs (and there are less in the display). If you try capturing it with a camera in slow motion you see odd artifacts, as segments light app in random order.

Were you able to identify the main microcontroller? I'm assumming it's OTP but we can dream. The 8 chip IC seems to be a 24Cxx memory, the pinout fits
